wgsl/index.bs

```diff
@@ -7450,7 +7457,9 @@ When several edges have to be created we use `X -> {Y, Z}` as a short-hand for `
       <td rowspan=2>*CF'*
       <td rowspan=2>
   <tr><td class="nowrap">var x = *e*;
-  </table>
+</table>
+
+Note: If the set of behaviors (see [[#behaviors]]) for an if, switch, or loop statement is {Next}, this means that we either did not diverge within the statement, or we reconverged, so we pick the node corresponding to control flow at the start of the statement as the node corresponding to control flow at the exit of the statement.
```

Comment:
This is a good explanation, and a tidy way to handle it.

Reply:
Thanks!